_rules/no-keyboard-trap-80af7b.md

```diff
@@ -48,31 +52,31 @@ _There are no major accessibility support issues known for this rule._
 
```

Comment:
Remove `## Aggregation Definition` and change the heading levels of Applicability / Expectation